src/TimeService.Business/Helpers/Workdays/WorkCalendarHelper.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+
+namespace LT.DigitalOffice.TimeService.Business.Helpers.Workdays
+{
+    public class WorkCalendarHelper
+    {
+        private const string BaseUrl = "https://isdayoff.ru/api/getdata";
+        private readonly HttpClient _httpClient;
+
+        public WorkCalendarHelper()
+        {
+            _httpClient = new HttpClient();
+        }
+
+        private string GetResponseContent(HttpResponseMessage responseMessage)
+        {
+            string stringResponse = responseMessage.Content.ReadAsStringAsync().Result;
+
+            if (stringResponse == "101")
```

Comment:
I suggest convert these magic numbers to constants

Reply:
it is api constants

Reply:
added consts

Reply:
i understand. but can we use constants instead hardcode?